docs/RISKS.md

```diff
@@ -0,0 +1,40 @@
+| 7 | Hardware problems | Hardware may fail or malfunction for any reason, including theft. Firmware or human failures may result in a lock-out of our developers so that the hardware becomes unusable for our project. | 0,2 | 9 | 1,8 | This risk can be mitigated by regular maintenance and testing. By making sure all important work is pushed to GitHub, we have another layer of security in case someone looses access to their laptop. | Jens |
+| 8 | Team members get ill or don't pass exams | At this big complexity in our project, we are in demand of working capacity. If fellow students (team members) are not able to work because they are no longer part of the university due to failing exams or getting ill for a longer period of time, the project may not reach its objectives. | 0,2 | 10 | 2 | This risk is hard to mitigate. All of us need to make sure to pass exams and pay attention to both physical and mental health. Further, cross-training of team members should remain a focus to make sure we don't get into knowledge silos. We are providing additional support, for example tutoring und mentoring not only for this project but overall to make sure everyone stays with us. Planning for team member absences, if necessary, helps as well. | Chris |
+| 9 | Mental overload | Mental overload can occur when team members are faced with a high degree of technical complexity in a project. Technical complexity refers to the level of difficulty in understanding and working with the technology or systems involved in the project. When technical complexity is high, team members may struggle to understand the requirements, design, or implementation of the project, leading to mental overload. For example, this could result in burnout or exhaustion by feeling out of control. | 0,3 | 7 | 2,1 | It is very important that every one of us receives adequate training. For example, tickets may include links to learning materials. The project manager takes care for every team member by encouraging to take regular breaks and time off to prevent stress and reduce burnout. Good project management makes sure timelines and expectations are realistic. | Jens |
+| 10 | Lack of motivation | Lack of motivation could result in destructive behavior, which may ultimately lead to project failure | 0,2 | 10 | 2 | Embracing to communicate openly is a significant part to tackle this. Celebrating successes, making sure clear goals and exectations are provided helps reduce the risk. | Chris |
```

Comment:
```suggestion
| 10 | Lack of motivation | Lack of motivation could result in destructive behavior, which may ultimately lead to project failure | 0,2 | 10 | 2 | Embracing to communicate openly is a significant part to tackle this. Celebrating successes, making sure clear goals and expectations are provided helps reduce the risk. | Chris |
```

maybe it would also help to ask the team members what kind of tasks they whish to do or what kind of features the app should provide in order to make them proud of their work.

Reply:
thank you :3 I'll add a sentence to write down your point